### PR TITLE
V1.5.5 release notes

### PR DESCRIPTION
**Pull request by stompsjo**
Release notes generated from `make_release_notes.sh` in `cyclus/release/utils`.

### CODE REVIEW

File: source/previous/index.rst [hunk 1]
```diff
@@ -28,6 +29,8 @@ have been included in the 1.5.0.
 ========= ========== ================ ==============
 Version   Date       Zip              Tar
 ========= ========== ================ ==============
+1.5.5     2019-01-14 `zip 1.5.5`_     `tar 1.5.5`_
+1.5.5     2019-01-14 `zip 1.5.5-rc1`_ `tar 1.5.5-rc1`_
```
Comment (bam241) [inline source/previous/index.rst]: probably don't need a rc1

File: source/previous/index.rst [hunk 2]
```diff
@@ -63,6 +66,10 @@ Version   Date       Zip              Tar
 0.1       2012-03-27 `zip 0.1`_       `tar 0.1`_
 ========= ========== ================ ==============
 
+.. _zip 1.5.5: https://github.com/cyclus/cyclus/archive/1.5.5zip
+.. _tar 1.5.5: https://github.com/cyclus/cyclus/arcive/1.5.5.tar.gz
+.. _zip 1.5.5-rc1: https://github.com/cyclus/cyclus/archive/1.5.5-rc1.zip
+.. _tar 1.5.5-rc1: https://github.com/cyclus/cyclus/archive/1.5.5-rc1.tar.gz
```
Comment (bam241) [inline source/previous/index.rst]: same no need for a rc1

**Comment by bam241** [comment]
LGTM otherwise thx @stompsjo

File: source/previous/release_notes.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+* Cymetric: 13 commits resulting in  7 files changed, 104 insertions(+), 4 deletions(-)
+
+Contributors
+============
+The following people contributed to this release of Cyclus.  A "*" by their
+name indicates a first time contributor.  Names follow alphabetically, 
+
+* Anna Caldwell-Overdier *
+* Baptiste Mouginot
+* gwenchee
```
Comment (nuclearkatie) [inline source/previous/release_notes.rst]: Is there a reason to have both Gwendolyn Chee and gwenchee?

Comment (stompsjo) [reply source/previous/release_notes.rst]:
```suggestion
```

Nope, I forgot to delete the username!

**Comment by nuclearkatie** [comment]
Looks good, thanks @stompsjo!

**Pull request merged**